src/components/HotspotEditorModal/HotspotEditorModal.test.jsx

```diff
@@ -241,9 +241,6 @@ describe('HotspotEditorModal', () => {
     // The initial dynamic hotspot title is showing
     const titleInputElement = screen.getByDisplayValue('dynamic test title');
 
-    // Click anywhere to remove focus the selected hotspot
```

Comment:
are these changes part of the carbon update?

Reply:
yes, they changed the text on their Tooltip component to Focus sentinel @davidicus